### DISCUSSION TITLE
How do I turn of jekyll?

### DISCUSSION THREAD

**Discussion by amenaijp** [community/community#23716] [Other Feature Feedback, Questions, & Ideas]
<p>Please explain in the most detail you can as I am very new to this. almost everywhere I go its “add a file called “.nojekyll”” but this doesn’t work for me, I am assuming since it is never specified that the file can be a .txt but it still doesn’t work. please help</p>

**Comment by FormerlyChucks** [comment]
<p>hello. you should be able to go into the settings of the repo, see a “theme chooser” and disable it.</p>

**Comment by amenaijp** [comment]
<p>Thats to turn off the theme, i mean turning off jekyll entirely so I can open folders through my page that start with dots or under scores as jekyll skips them in the building process when my websites viewed</p>

**Comment by Simran-B** [comment]
<aside class="quote no-group" data-username="AmethystEdits" data-post="1" data-topic="138373">
<div class="title">
<div class="quote-controls"></div>
 AmethystEdits:</div>
<blockquote>
<p>since it is never specified that the file can be a .txt</p>
</blockquote>
</aside>
<p>I don’t understand what you mean. The exact file name has to be <code>.nojekyll</code>, not <code>.nojekyll.txt</code>. It’s not expected to have any content, but it could have any content, as merely its presence is checked to determine if Jekyll is supposed to be turned off.</p>

**Comment by amenaijp** [comment]
<p>I mean no matter what type of file i put there called <code>.nojekyll</code> jekyll still does jekyll things so I cant figure out how to turn it off, sorry for any confusion</p>

**Comment by amenaijp** [comment, marked as answer]
Figured it out; open your repo that has the site stuff, same branch, and click on the “new file” thing, not from the desktop app. then type the name as .nojekyll and commit it.

**Reply by limalexandre** [reply]
I did as you said, added an empty file called ".nojekyll" inside my "node-modules" folder. It solved my problem with the error: "Failed to load resource: Server responded with a status of 404'" for some files. Thank you very much!

**Comment by personalsoft-fabianobonin** [comment]
Did you solve this? I could not disable jekyll by no means, neither using .jekyll file or _config.yml in project root.

**Reply by Simran-B** [reply]
It's a bit different nowadays:

> GitHub Pages now uses GitHub Actions to execute the Jekyll build. [...] if GitHub Actions is unavailable or disabled, adding a .nojekyll file to the root of your source branch will bypass the Jekyll build process

https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/about-github-pages-and-jekyll

You may want to check the GitHub Actions configuration respectively for any workflows that might be in place and cause Jekyll to build your source files.
